benchmark/src/jmh/kotlin/kotlinx/benchmarks/json/TwitterFeedStreamBenchmark.kt

```diff
@@ -0,0 +1,84 @@
+package kotlinx.benchmarks.json
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import kotlinx.benchmarks.model.MacroTwitterFeed
+import kotlinx.benchmarks.model.MicroTwitterFeed
+import kotlinx.serialization.json.*
+import org.openjdk.jmh.annotations.*
+import java.nio.file.Files
+import java.nio.file.Path
+import java.util.concurrent.TimeUnit
+import kotlin.io.path.deleteIfExists
+import kotlin.io.path.outputStream
+
+@Warmup(iterations = 7, time = 1)
+@Measurement(iterations = 7, time = 1)
+@BenchmarkMode(Mode.Throughput)
+@OutputTimeUnit(TimeUnit.SECONDS)
+@State(Scope.Benchmark)
+@Fork(2)
+open class TwitterFeedStreamBenchmark {
```

Comment:
I see that a lot of `decode*` benchmarks dropped their score by ~15%.

It seems to be a serious hit and it's definitely worth pinpointing it before merging this PR, so we are sure we cannot avoid it (or can and then will do)